.packit.yaml

```
@@ -65,3 +81,5 @@
       - rhel-9-x86_64
       - epel-9-aarch64
       - epel-9-x86_64
+    packages:
+      - flightctl
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)